multichain-testing/scripts/install.sh

```diff
@@ -24,7 +24,7 @@ NAMESPACE=""
 HELM_REPO="starship"
 HELM_CHART="starship/devnet"
 HELM_REPO_URL="https://cosmology-tech.github.io/starship/"
-HELM_CHART_VERSION="0.2.2"
+HELM_CHART_VERSION="0.2.8"
```

Comment:
This release also contains a fix for: https://github.com/cosmology-tech/starship/issues/482

```diff
diff --git a/multichain-testing/config.yaml b/multichain-testing/config.yaml
index a82dbac5e..320274650 100644
--- a/multichain-testing/config.yaml
+++ b/multichain-testing/config.yaml
@@ -71,6 +71,7 @@ chains:
               allow_messages: ['*']
     faucet:
       enabled: true
+      type: starship
     ports:
       rest: 1314
       rpc: 26654
```

```diff
diff --git a/multichain-testing/test/stake-ica.test.ts b/multichain-testing/test/stake-ica.test.ts
index 06e36bbc6..56d064f57 100644
--- a/multichain-testing/test/stake-ica.test.ts
+++ b/multichain-testing/test/stake-ica.test.ts
@@ -103,13 +103,6 @@ const stakeScenario = test.macro(async (t, scenario: StakeIcaScenario) => {
     `address for ${scenario.chain} is valid`,
   );

-  if (scenario.chain === 'cosmoshub') {
-    // see https://github.com/cosmos/cosmjs/pull/1593 for upstream fix
-    t.pass(
-      `SKIP ${scenario.chain}. @cosmjs/faucet does not support ICA address length.`,
-    );
-    return;
-  }
   const { creditFromFaucet, getRestEndpoint } = useChain(scenario.chain);
   const queryClient = makeQueryClient(getRestEndpoint());
```

We should also be able to make the above changes, and test the cosmoshub staking flow. I was running into issues on my local a ~week ago and haven't had time to investigate further.

The starship env was also running slower for me on this revision (was hitting CPU limits I haven't observed before), so I am cautious of immediately approving. Curious to hear how your experience has running this (and the revision we have on master locally).

Reply:
Seems my computer was having a bad day 🤷‍♂️